CMakeLists.txt

```diff
@@ -553,9 +553,33 @@ ELSE (WITH_BUNDLED_SSL)
     ENDIF (OPENSSL_FOUND)
 ENDIF (WITH_BUNDLED_SSL)
 
+include(ExternalProject)
+
+IF (WITH_MRUBY)
+    SET(H2GET_MRUBY_OPTS "-DWITH_MRUBY_LIB=${CMAKE_CURRENT_BINARY_DIR}/mruby/host/lib/libmruby.a -DWITH_MRUBY_INCLUDE=${CMAKE_CURRENT_SOURCE_DIR}/deps/mruby/include")
+ENDIF ()
```

Comment:
Do we need this `IF`?

I assume that we are building h2get regardless of if mruby support in on for h2o. If that is the case, it might make things a bit simpler if we use the mruby bundled with h2get (we do, right?) unconditionally.